CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 ### Mimirtool
 
 * [BUGFIX] Fix out of bounds error on export with large timespans and/or series count. #5700
+* [BUGFIX] Update Grafana analysis timeout behavior by creating child contexts for each dashboard request. #5915
```

Comment:
Could you please describe this in terms of the user impact? eg. "Fix issue where `--read-timeout` was applied to entire `mimirtool analyze grafana` invocation rather than individual Grafana API calls"

Reply:
Addressed. Thank you!